### PR TITLE
Drop built-in dapps for stable (binary size)

### PR DESCRIPTION
**Pull request by jacogr**
Tests locally -

```
Pre : 89474728
Post: 83532768
```

Be aware, numbers above is with `--no-default-features` flags, so decrease is important, however overall base size in this config is much larger than done with defaults. Final sizes only checked once there is a pre-compiled build, i.e. merge

### CODE REVIEW

File: js/src/Dapps/blacklist.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = [
+  // tokendeploy
+  '0xf9f2d620c2e08f83e45555247146c62185e4ab7cf82a4b9002a265a0d020348f',
```
Comment (debris) [inline js/src/Dapps/blacklist.js]: where are these hashes coming from?

Comment (jacogr) [reply js/src/Dapps/blacklist.js]: They are the actual ids are registered in the registries. (Initially created using the sha3 of the name, same across all chains)

**Comment by kirushik** [comment]
Are those measurements in bytes?
Any expectation on how that would translate in the actual binary size -- since obviously we have some layers of compression before embedding resources in the final released binary.

**Comment by jacogr** [comment]
In bytes yes, on Mac OS X.

I need to build with `--no-default-features --features ui` to pull in the changes locally since there is no pre-compiled (yet), would love to be more accurate. :(

However, the JS itself probably won't compress, it is run through Uglify already.

**Pull request merged**